presto-main/src/main/java/com/facebook/presto/sql/gen/ByteCodeUtils.java

```diff
@@ -282,7 +283,12 @@ public static ByteCodeNode generateWrite(CallSiteBinder callSiteBinder, Scope sc
                     .invokeInterface(BlockBuilder.class, "appendNull", BlockBuilder.class)
                     .pop();
         }
-        String methodName = "write" + Primitives.wrap(type.getJavaType()).getSimpleName();
+
+        Class valueJavaType = type.getJavaType();
```

Comment:
`Class<?>` don't use raw generics. there's an IntelliJ warnings for this you should turn on